artview/components/plot_radar.py

```diff
@@ -778,9 +778,10 @@ def _check_file_type(self):
     ########################
     def _quick_savefile(self, PTYPE=IMAGE_EXT):
         '''Save the current display via PyArt interface.'''
-        PNAME = self.display.generate_filename(
+        path = self.display.generate_filename(
             self.Vfield.value, self.Vtilt.value, ext=IMAGE_EXT)
-        print "Creating " + PNAME
+        self.canvas.print_figure(path, dpi=DPI)
+        self.statusbar.showMessage('Saved to %s' % path)
```

Comment:
Glad you did this, I was making the same change, but I'll wait to address any other until we merge this.